### PR TITLE
Add Travis configuration

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

* Add Travis build configuration
* Add Travis badge to the README
* Exclude (non-existing) integration tests from jacoco analysis (otherwise the build would fail)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  directories:
+  - $HOME/.gradle/caches/
+  - $HOME/.gradle/wrapper/
+
+before_install:
+  - curl https://raw.githubusercontent.com/hmcts/reform-api-docs/master/bin/publish-swagger-docs.sh > publish-swagger-docs.sh
+
+script:
+  - ./gradlew test
+  - ./gradlew integration
```
Comment (nitinprabhu) [inline .travis.yml]: can be added later ?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+language: java
+
+branches:
+  only:
+    - master
```
Comment (timja) [inline .travis.yml]: why only run on master?

Comment (doncem) [reply .travis.yml]: running only on master based branches only. including master. so one can push as many commits as he likes and travis will be executed only when pr is raised

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
-# bulk-scan-orchestrator
\ No newline at end of file
+# bulk-scan-orchestrator
```
Comment (doncem) [inline README.md]: Perhaps it is a good occasion to at least name it `Bulk Scan Orchestrator` 😉

File: build.gradle [hunk 4]
```diff
@@ -80,7 +80,8 @@ pmd {
 }
 
 jacocoTestReport {
-  executionData(test, integration)
+  // TODO: add 'integration' when integration tests are added
+  executionData(test)
```
Comment (doncem) [inline build.gradle]: the integration task is executed in travis build so it is no harm in leaving execution data as it was

File: .travis.yml (hunk 1 above)
Comment (lgonczar) [reply .travis.yml]: The task exists already, so I think it's fine that it's included. We don't risk forgetting to add it later.

File: README.md (hunk 3 above)
Comment (lgonczar) [reply README.md]: I will do the overall readme part in one of the coming PRs.

File: build.gradle (hunk 4 above)
Comment (lgonczar) [reply build.gradle]: That task is included, but even if it wasn't, without that bit may cause `jacocoTestReport` issues locally until integration tests are created.

**Pull request merged**